lib/helpers.js

```diff
@@ -311,11 +315,15 @@ function _getExpectedRootCapability({invocationTarget, edvId}) {
   return invocationTarget;
 }
 
-// Note: for dereferencing `did:key` URLs
+// Note: for dereferencing `did:` URLs
 async function _documentLoader(url) {
-  // TODO: move to did-key lib
-  if(url.startsWith('did:key:')) {
-    const document = await didKeyDriver.get({url});
+  let document;
+  if(url.startsWith('did:')) {
+    document = await didIo.get({did: url});
+    // FIXME
```

Comment:
What is this FIXME about?

Reply:
Just a reminder to replace lines 324-326 (since the return signature of `didIo.get()` will be changing.

Reply:
Not addressing this now (will address once did-io is refactored).